includes/class-woothemes-sensei-course.php

```diff
@@ -2020,4 +2033,148 @@ public static function the_course_action_buttons( $course ){
 
     }// end the_course_action_buttons
 
+    /**
+     * function user_profile_show_courses
+     *
+     * add Course checkboxes to Add New User and
+     * Edit User screens.
+     *
+     * @since 1.9.0
+     * @param object $user
+     * @return void
+     */
+
+    public function user_profile_show_courses ( $user ) {
+
+        if (!current_user_can('manage_options')) {
+
+            return;
+
+        }
+
+        ?>
+
+        <h3>Sensei</h3>
+        <span><strong>NOTE: unticking courses here will remove this user from those courses and reset their progress</strong>.</span>
+
+
+        <table class="form-table">
+
+            <tr>
+                <th><label for="company">Courses</label></th>
+                <td>
+
+        <?php
+
+        $post_args = array(
+            'posts_per_page'   => -1,
```

Comment:
@mahangu  we should always aim to place an upper limit, -1 will break sites with thousands of courses.

This brings me to another thought, what if a client has 1000 courses? This interface would simply not work. Something like this will be better: http://jsfiddle.net/X6V2s/66/ (just the idea).

This will only go into 2.0 so we have to to come up with a more scalable solution.